roles/ispmail-certificate/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: update ca-certificates
+  command: /usr/sbin/update-ca-certificates
+  delegate_to: "{{ item }}"
+  with_inventory_hostnames:
```

Comment:
Is this necessary, considering that this handler is called by a task that already has this `with_inventory_hostnames` statement ?

Reply:
Yes, `with_inventory_hostnames` is just a shorthand for loop over `hosts["mygroup"]`